bp-esa-frontend/src/components/reportPage.js

```diff
@@ -12,40 +13,23 @@ class ReportPage extends Component {
       { name: 'removal', display: 'Removal'},
```

Comment:
Use the same function I talked about above to deal with the repetition here as well.